### ISSUE TITLE
Circular dependency

### ISSUE THREAD

**Issue by FredPraca** [AdaCore/gtkada#13]
Hello,
I'm trying to update ports on FreeBSD but I run into a chicken-egg problem.
To build gtkada reference manual, I need gnatdoc which is part of GPS.
The problem is that GPS requires gtkada to be installed.
So, to build gtkada reference manual, I would need a kind of gtkada bootstrap library to build gps and then build gtkada reference manual.

Should not gnatdoc be a project on its own ?

**Comment by setton** [comment]
Yes this is unfortunate, and yes the plan is for gnatdoc to become its own project, when we have switched it to use [libadalang](https://github.com/AdaCore/libadalang) as the engine for cross-references, rather than the GPS engine...

**Comment by FredPraca** [comment]
Thanks for your quick answer Nicolas. 
So for the moment, I will just disable the build for the reference manual.

**Comment by setton** [comment]
This is a reasonable workaround for now, and yes I propose to keep the issue open until we have resolved it.
